projects/nerveswap/index.js

```diff
@@ -1,8 +1,23 @@
 const utils = require('../helper/utils');
 const { BigNumber } = require("bignumber.js");
+const baseURL = "https://api.swap.nerve.network/swap";
 
 async function fetch() {
-  let total = await utils.fetchURL('https://api.swap.nerve.network/swap/total/info');
+  let total = await utils.fetchURL(baseURL + '/total/info');
+  let tvl = new BigNumber(total.data.data.tvl);
+  tvl = tvl.shiftedBy(-18).toFixed(6);
+  return tvl;
+}
+
+async function nuls() {
+  let total = await utils.fetchURL(baseURL + '/nuls/info');
+  let tvl = new BigNumber(total.data.data.tvl);
+  tvl = tvl.shiftedBy(-18).toFixed(6);
+  return tvl;
+}
+
+async function kava() {
+  let total = await utils.fetchURL(baseURL + '/kava/info');
```

Comment:
can you fetch kava data on chain? is this a uniswap fork? if so, can you share the factory address

Reply:
This pool's data cannot be fetched from the Kava chain as it is on the bridge network, which is not based on EVM. Currently, Kava assets have been cross-chained to the bridge network. Since the Kava assets on the cross-chain bridge cannot be calculated on the Kava chain, we will calculate the Kava assets on the bridge network to the Kava chain. Is this acceptable?

Reply:
by bridge network, you mean nerve? if that is the case, I dont think we can count it as kava tvl